### PR TITLE
Add Carolynshin southsanfrancisco

### PR DESCRIPTION
**Pull request by carolynshin**
I added restaurants and things to do. 

@hollenberry, have you been to South San Francisco? Is there anything I should add?

### CODE REVIEW

File: carolynshin-southsanfrancisco.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### South San Francisco
+
+#### Things to eat :fork_and_knife_with_plate:
+- [Toppings](https://www.yelp.com/biz/toppings-restaurant-south-san-francisco)
+- [Fil-Am Cuisine 2](https://www.yelp.com/biz/fil-am-cuisine-2-south-san-francisco?osq=dinner)
+
+#### Things to do :eyes:
+- Play Pokémon Go
```
Comment (tgruenewald) [inline carolynshin-southsanfrancisco.md:8]: Gotta catch them all!

**Comment by tgruenewald** [comment]
Hi @carolynshin , can you comment on my pull request?

**Comment by carolynshin** [comment]
@tgruenewald, I sure can!

**Comment by hollenberry** [comment]
@carolynshin I haven't been to South San Francisco before! Do you have any recommendations for Sushi?

Also, where's the best place to play Pokemon Go?

**Pull request merged**